gamemodes/terrortown/gamemode/client/cl_radio.lua

```diff
@@ -273,21 +273,21 @@ end
 -- this way players can bind keys to them
 local function RadioCommand(ply, cmd, arg)
 	if not IsValid(ply) then
-		print("ttt_radio failed, invalid player")
+		ErrorNoHaltWithStack("ttt_radio failed, invalid player")
 
 		return
 	end
 
 	---
 	-- @realm client
 	if hook.Run("TTT2ClientRadioCommand", cmd) then
-		print("ttt_radio, execution prevented by a hook")
+		ErrorNoHaltWithStack("ttt_radio, execution prevented by a hook")
```

Comment:
no, this is bad. If a hook prevents the radio, it shouldn't throw an error. But tbh, this print should probably be removed or moved to a MStack message

Reply:
If this should be moved to a MStack message this would also need a lang string.

i'd say reverting this for now matches the scope of this PR.

Reply:
or put it as a DEV(1 for now